code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3467,6 +3467,72 @@
 	glass_name = "glass of Winter Offensive"
 	glass_desc = "Proven to be more successful than the campaign."
 
+// Skrellian drinks
+//====================
+// Some are alocholic, some are not
+
+/datum/reagent/alcohol/ethanol/thirdincident
+	name = "The Third Incident"
+	id = "thirdincident"
+	color = "#1936a0"
+	strength = 10
+	description = "A controversial drink popular with the punk youth of the Jargon Federation. Represents blood, eggs, and tears."
+	taste_description = "genophage sadness"
+
+	glass_icon_state = "thirdincident"
+	glass_name = "glass of the Third Incident"
+	glass_desc = "A controversial drink popular with the punk youth of the Jargon Federation. Represents blood, eggs, and tears."
+
+/datum/reagent/drink/upsidedowncup
+	name = "Upside-Down Cup"
+	id = "upsidedowncup"
+	color = "#B2110A"
+	description = "The classic Skrell meme."
```

Comment:
I wonder if this is a bit too on the nose?

Reply:
That's the point, no?

Reply:
It was my intention. Every joke they have is memetic. This is one even small tadpoles are exposed to. Consider it a set-up like "Why did the chicken cross the road?" Who even likes those jokes? Skrell would; to the point of naming fun and fizzy drinks after it to spread awareness.

(And more importantly, Skrell don't have noses.)

Reply:
I agree with skull, replace meme with joke.

Reply:
Sorry you are destroyed with lore

![image](https://user-images.githubusercontent.com/17379794/51629912-d5bd1580-1f59-11e9-9b1b-8d56082e0b39.png)